README.md

```diff
@@ -2,6 +2,569 @@
 
 > Expressions language for Clojure(Script) and JavaScript inspired by Microsoft Excel ™
 
+# Rationale
+
+In some applications, the lion's share of business logic is concentrated in dynamic expressions. Often they are not even part of the code base and are stored in a database, files, or somewhere in the cloud. And sometimes these expressions need to be accessed not by programmers. For example, managers or operators of call centers. Making them learn clojure or python will not be fair. This library is designed to combine the best of two completely different worlds: the most understandable syntax from Excel and the speed of Clojure.
```

Comment:
"need to be accessed by _non-technical personnel, like_ managers or call center operators. Making them learn clojure or python _is_ not fair."